data-upgrade-news/src/main/java/org/exoplatform/news/upgrade/ContentArticlePropertiesUpgrade.java

```diff
@@ -0,0 +1,195 @@
+                                         SpaceService spaceService,
+                                         UserACL userACL) {
+    super(initParams);
+    this.noteService = noteService;
+    this.metadataService = metadataService;
+    this.identityManager = identityManager;
+    this.spaceService = spaceService;
+    this.userACL = userACL;
+  }
+
+  @Override
+  public void processUpgrade(String oldVersion, String newVersion) {
+    long startupTime = System.currentTimeMillis();
+    LOG.info("Start upgrade of content page properties");
+    int notMigratedContentPagesPropertiesCount;
+    int processedContentPagesPropertiesCount = 0;
+    int totalContentPagesPropertiesCount = 0;
+    try {
+      MetadataFilter metadataFilter = getMetadataFilter();
+      metadataFilter.setMetadataObjectTypes(List.of(NEWS_METADATA_PAGE_VERSION_OBJECT_TYPE,
+                                                    NEWS_METADATA_DRAFT_OBJECT_TYPE,
+                                                    NEWS_METADATA_LATEST_DRAFT_OBJECT_TYPE));
+      List<MetadataItem> metadataItems = metadataService.getMetadataItemsByFilter(metadataFilter, 0, 0);
+      totalContentPagesPropertiesCount = metadataItems.size();
+      for (MetadataItem metadataItem : metadataItems) {
+        if (metadataItem != null && !MapUtils.isEmpty(metadataItem.getProperties())) {
+          Map<String, String> contentProperties = metadataItem.getProperties();
+          Page page = null;
+          String objectType = NOTE_METADATA_PAGE_OBJECT_TYPE;
+          if (metadataItem.getObjectType().equals(NEWS_METADATA_PAGE_VERSION_OBJECT_TYPE)) {
+            PageVersion pageVersion = noteService.getPageVersionById(Long.valueOf(metadataItem.getObjectId()));
+            if (pageVersion != null && pageVersion.getParent() != null) {
+              page = pageVersion.getParent();
+            }
+          } else {
+            page = noteService.getDraftNoteById(metadataItem.getObjectId(), userACL.getSuperUser());
+            objectType = NOTE_METADATA_DRAFT_PAGE_OBJECT_TYPE;
+          }
+          if (page != null) {
+            NoteMetadataObject noteMetadataObject = buildNoteMetadataObject(page, null, objectType);
+            MetadataItem noteMetadataItem = getNoteMetadataItem(page, null, objectType);
+            Map<String, String> noteProperties = new HashMap<>();
+            long creatorId = Long.parseLong(identityManager.getOrCreateUserIdentity(page.getAuthor()).getId());
+
+            if (noteMetadataItem != null && noteMetadataItem.getProperties() != null) {
+              noteProperties = noteMetadataItem.getProperties();
+            }
+            if (noteProperties.getOrDefault(CONTENT_ILLUSTRATION_ID, null) != null) {
+              noteProperties.put(FEATURED_IMAGE_ID, contentProperties.get(CONTENT_ILLUSTRATION_ID));
+              noteProperties.put(FEATURED_IMAGE_UPDATED_DATE, String.valueOf(new Date().getTime()));
```

Comment:
IMO if we do have the latest update date, we use it otherwise we can set the current timestamp instead

Reply:
IMO, it's not a big deal, in current case we will just re-cash the saved illustrations as they have a new lastUpdated.
the `lastUpdated` date exists only in the `fileInfo` of the `fileItem`
and we will need to use `filService.get() `+ checks and process  to get the `lastUpdated`

but if you want to use it, i will update the PR.